README.md

```diff
@@ -1,40 +1,47 @@
+LAOS is the Universal layer 1 for digital assets across all blockchains, introducing unique features such as bridgeless minting and evolution. 
 
-> **NOTE**: To streamline our development process and encourage diverse contributions, we've consolidated three core projects—Ownership Chain, Evolution Chain, and Bridge—into this single monorepo.
+![LAOS Logo](docs/LAOS_logo.png)
 
-## Introduction
+LAOS's main documentation can be found here:
+- **[LAOS Whitepaper](https://github.com/freeverseio/laos-whitepaper/blob/main/laos.pdf)**, for an in-depth, fully detailed description.
+- **[LAOS Litepaper](https://spaces.gorengine.com/laos/LAOS_litepaper.pdf)**, for a high-level, use case oriented description.
+- **[SBP Roadmap](https://github.com/freeverseio/laos-roadmap)**, the engineering roadmap within the Substrate Builders Program, ending on Q1 2024.
 
-The LAOS Monorepo is the centralized codebase for LAOS, a next-generation, non-custodial, dynamic NFT (Non-Fungible Token) protocol. With the aim of redefining the NFT landscape, LAOS introduces unique features such as universal minting and dynamic asset evolution. This monorepo houses all the essential resources and documentation needed for building and deploying assets on the LAOS blockchain platform.
+LAOS is fully open source and decentralized. The main code repositories are:
 
-## Core Projects
+* **[The LAOS Parachain](#the-laos-parachain-monorepo)**. You are already in it. To streamline the development process and encourage diverse contributions, all core components of the Parachain were integrated in this single monorepo.
 
-### [Ownership Chain](./ownership-chain/)
+* **[The Universal Node](https://github.com/freeverseio/laos-universal-node)**. This node streamlines the integration process for DApps aiming to incorporate bridgeless minting and evolution across various chains, including Ethereum, by merely adjusting the RPC endpoint to connect to the relevant Universal Nodes.
 
-Ownership Chain is dedicated to the enforcement and management of asset ownership within the LAOS ecosystem. It ensures secure, transparent, and efficient tracking of asset ownership changes, serving as a cornerstone of the LAOS platform.
+* **[The Universal ERC721 Solidity Templates](https://github.com/freeverseio/laos-erc721)**. This template contains the minimal extensions of the ERC721 OpenZeppelin implementation that enables bridgeless minting and evolution in any EVM chain.
 
-### [Evolution Chain](./evolution-chain/)
+LAOS is targeting late Q1'2024 to bid for a Parachain slot on Polkadot.
+Presently, LAOS is accessible through its test network, named **Caladan**. Additionally, it is in the process of participating in a Parachain auction on Kusama; the Parachain will be named **KLAOS**, which stands for **K**usama LAOS. 
```

Comment:
Should the last LAOS be bold? So the full text reads **K**usama **LAOS**.

Reply:
no need, thanks!